lib/prolongator.cu

```diff
@@ -234,55 +234,86 @@ namespace quda {
     if (out.Ncolor() == 3) {
       const int fineColor = 3;
       if (nVec == 4) {
-	Prolongate<Float,fineSpin,fineColor,coarseSpin,4>(out, in, v, fine_to_coarse, parity);
+        Prolongate<Float,fineSpin,fineColor,coarseSpin,4>(out, in, v, fine_to_coarse, parity);
       } else if (nVec == 6) { // Free field Wilson
-  Prolongate<Float,fineSpin,fineColor,coarseSpin,6>(out, in, v, fine_to_coarse, parity);
+        Prolongate<Float,fineSpin,fineColor,coarseSpin,6>(out, in, v, fine_to_coarse, parity);
       } else if (nVec == 24) {
-	Prolongate<Float,fineSpin,fineColor,coarseSpin,24>(out, in, v, fine_to_coarse, parity);
+        Prolongate<Float,fineSpin,fineColor,coarseSpin,24>(out, in, v, fine_to_coarse, parity);
+#ifdef GPU_WILSON_DIRAC
```

Comment:
Should be `NSPIN4` to ensure non-Wilson Dirac operator MG still works

Reply:
Done.